src/ngraph/partial_shape.hpp

```diff
@@ -305,4 +305,6 @@ namespace ngraph
         static constexpr DiscreteTypeInfo type_info{"AttributeAdapter<PartialShape>", 0};
         const DiscreteTypeInfo& get_type_info() const override { return type_info; }
     };
+
+    inline bool is_scalar(const PartialShape& shape) { return shape.rank() == 0; }
```

Comment:
Can you use [this](https://github.com/NervanaSystems/ngraph/blob/7bb94ca049e720c35adb5844e7deb6c93ddf1b49/src/ngraph/shape.hpp#L98) instead? Or even remove this method since I don't see any references to it.

Reply:
I can't because `PartialShape` class doesn't have a `size()` method.